README.md

```diff
@@ -49,17 +49,19 @@ navigator.VuforiaPlugin.startVuforia(
 );
 ```
 
-**NOTE**: You will need to replace `YOUR_VUFORIA_KEY` with a valid license key for the plugin to launch correctly.
-
-For testing you can use the `targets/PluginTest_Targets.pdf` file, it contains all four testing targets.
-
+**NOTES**: 
+* You will need to replace `YOUR_VUFORIA_KEY` with a valid license key for the plugin to launch correctly.
+* For testing you can use the `targets/PluginTest_Targets.pdf` file inside the plugin folder; it contains all four testing targets.
 
 #### Using your own data
 We know that eventually you're going to want to use your own data. To do so, follow these extra steps.
 
 ##### `www/`
 First, create a `targets/` folder inside `www/` and place your own `.xml`/`.dat`/`.pdf` files inside.
 
+##### `.js`
+You will need to replace the `PluginTest.xml` argument with `www/targets/CustomData.xml` and the `[ 'logo', 'iceland', 'canterbury-grass', 'brick-lane' ]` argument with the specific information for your application. These are here for initial setup and demo purposes only.
```

Comment:
Does it have to be CustomData.xml? Not sure if I'm just splitting hairs at this  point 😜